tasks/main.yml

```diff
@@ -23,6 +23,14 @@
     - "{{ spark_log_dir }}"
     - "{{ spark_run_dir }}"
 
+- name: Ensure Spark download and install directories exist
```

Comment:
Yes, I think it is slightly cleaner if the two are different tasks. My next commit also gives write access to the ```spark``` group (such that other users can start and stop the cluster), which I guess we wouldn't want to be the case for the code.